package.json

```diff
@@ -83,11 +83,11 @@
 		"build-languages-if-enabled": "node -e \"(process.env.BUILD_TRANSLATION_CHUNKS === 'true' || process.env.ENABLE_FEATURES === 'use-translation-chunks') && process.exit(1)\" || yarn run build-languages",
 		"calypso-doctor": "./node_modules/.bin/calypso-doctor",
 		"clean": "yarn run clean:packages && yarn run clean:build && yarn run clean:public && yarn run clean:translations",
-		"clean:build": "npx rimraf build 'client/server/bundler/*.json' client/server/devdocs/search-index.js",
+		"clean:build": "rm -rf build client/server/bundler/*.json client/server/devdocs/search-index.js || true",
 		"clean:packages": "yarn workspaces foreach --all --parallel --verbose --exclude 'wp-calypso' run clean",
-		"clean:public": "npx rimraf public",
-		"clean:translations": "npx rimraf build/strings calypso-strings.pot 'chunks-map.*.json'",
-		"distclean": "yarn run clean && npx rimraf '**/node_modules' '**/.cache' .yarn/install-state.gz",
+		"clean:public": "rm -rf public",
```

Comment:
Interesting that the yarn shell doesn't mind when the `public` directory doesn't exist, but would complain if there was glob like `rm -rf publ?c`. Somewhat inconsistent IMO, but that's how it works.